src/pages/debrief/cat-cpc/components/debrief-cpc-card/debrief-cpc-card.scss

```diff
@@ -0,0 +1,12 @@
+debrief-cpc-card {
+  .cpc-line-break {
+    border-bottom: 1px solid;
+    border-bottom-color: #9b9b9b;
```

Comment:
isn't there an existing sass variable we can use for this? assume similar is done elsewhere?